Scripts/AutoApplyDrivers/MODULE_Functions/MODULE_Drivers.psm1

```diff
@@ -195,7 +195,15 @@ function Query-DriverListAgainstOnlineOS
 
     try
     {
+        try
```

Comment:
This is redundant as if `Get-WindowsDriver` fails we skip immediately to the catch.  We are very unlikely to fail the code below this as it's just simple queries against the data returned (unless the data returned is bad).  99% likely this is where we fail.  I would yank out this try/catch, and see the suggested change to like 237 below.

Reply:
I wanted to avoid the `Get-WinodwsDriver is not a valid function, blah blah blah` exception being thrown here as it clutters up the log.